### DISCUSSION TITLE
Content type access rights

### DISCUSSION THREAD

**Discussion by xperiandri** [OrchardCMS/OrchardCore#8233] [Q&A]
Is it possible to set access by role to a particular content type?

**Comment by hishamco** [comment]
Why not, you can use `IAuthroizationService` to authorize your content to be accessed via a specific role, I'm not sure there's a bulit-in function but it's doable

**Reply by hishamco** [reply]
This is https://github.com/OrchardCMS/OrchardCore/blob/1879619df4418ff175a59ba70dcb36bab89fcbb5/src/OrchardCore/OrchardCore.Infrastructure/Security/AuthorizationServiceExtensions.cs how we could authorize a user with a specific object - content type in your case - you could do something similar with roles

**Comment by jtkech** [comment]
Already implemented, at least with the dev branch, when editing a role, there are global permissions tied to the `OrchardCore.Contents` feature, but also specific permissions for each content types

**Reply by hishamco** [reply]
I think this show up if we mark the content type as `Securable`, Am I right?

**Comment by jtkech** [comment]
Yes ;)

**Comment by xperiandri** [comment]
@jtkech we have found those settings. For the anonymous role, all checkboxes are unchecked for the content type we want to make a restriction.
But if we open it in the anonymous browser tab using a link it opens.
